README.md

```diff
@@ -1,73 +1,122 @@
 
+### 3 . Setting VAPID Keys
+
+Run the following command to generate Public and Private VAPID :
+
+```bash
+yarn web-push-generate
 ```
-yarn chain
+
+- Set `PUBLIC_KEY_VAPID` and `PRIVATE_KEY_VAPID` variables in `packages/nextjs/.env` file.
+- Set `NEXT_PUBLIC_PUBLIC_KEY_VAPID` in `packages/nextjs/.env.local` file.
+
+### 4 . Starting the app
+
+```bash
+yarn start
 ```
 
-This command starts a local Ethereum network using Hardhat. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in `hardhat.config.ts`.
+Visit your app on: `http://localhost:3000`.
+
+> Note: You can disable dev server logs by uncommenting line `disable: process.env.NODE_ENV=== "development"` in `packages/nextjs/next.config.mjs` file.
 
-3. On a second terminal, deploy the test contract:
+### 5 . Testing notification on local
+
+1. Install the PWA from Chrome web browser.
+
+2. Open the PWA, click _"Allow Notification"_ button => This will ask for permission & register the subscription in DB.
+
+3. Once its successful you will see _"Notify All"_ button => This button makes "POST" request to `packages/nextjs/pages/api/push/notify-all.ts` which will send notification to all the subscribers
+
+### 6 . Deploying your PWA to Vercel
+
+To deploy directly from the CLI, run this and follow the steps to deploy to Vercel:
 
 ```
-yarn deploy
+yarn vercel
 ```
 
-This command deploys a test smart contract to the local network. The contract is located in `packages/hardhat/contracts` and can be modified to suit your needs. The `yarn deploy` command uses the deploy script located in `packages/hardhat/deploy` to deploy the contract to the network. You can also customize the deploy script.
+Once you log in (email, github, etc), the default options should work. It'll give you a public URL.
 
-4. On a third terminal, start your NextJS app:
+If you want to redeploy to the same production URL you can run:
 
 ```
-yarn start
+yarn vercel --prod
 ```
 
-Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the contract component or the example ui in the frontend. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
+#### 6.1 . Setting Environment Variables
+
+When you deploy to Vercel you have to set all the environment variables from your `packages/nextjs/.env` and `packages/nextjs/.env.local` files into your Vercel Environment Variables section.
+
+You can do this in the Vercel Project dashboard under _"Settings > Environment Variables"_.
+
+> Hint: You can mass copy all the config variables from your`packages/nextjs/.env` and `packages/nextjs/.env.local` config files and paste them into the Vercel form.
+
+![VercelEnvironmentVariables](https://github.com/BuidlGuidl/PWA-burner-wallet/assets/55535804/8d1a56cf-a7c0-4ebe-8949-18673d6542e7)
```

Comment:
This is the only screenshot we have in the README... I think we don't need it.

Reply:
Agree! I had the same feeling watching the lonely screenshot in the Readme, but my mind convinced me that we would have more screenshots soon so it would be fixed. But probably that screenshot is not even needed, and right now looks bad.

Deleted! 🙌